src/Components/Buttons/GetStarted.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Header, Table, Icon, Button } from 'semantic-ui-react'
+
+//onClick={this.redirect('/start') basically linking to the start spage.
+//maybe there's a better way than using this one-line crap
+
+const GetStarted = () => (
+  <div className='ui center aligned basic segment'>
+
+      <Button padded centered size='huge' className='centered'>
```

Comment:
padded  and centered  on the button component gives a warning when compiled.
I don't think those values can be associated with the semantic button buttons are always padded if the  compact attribute is not set.